wpiutil/src/main/native/include/support/SafeThread.h

```diff
@@ -8,7 +8,8 @@
 #pragma once
 
 #ifdef _MSC_VER
-#pragma message "warning: support/SafeThread.h is deprecated; include wpi/SafeThread.h instead"
+#pragma message \
+    "warning: support/SafeThread.h is deprecated; include wpi/SafeThread.h instead"
```

Comment:
With a linebreak between "#pragma message" and the rest of the line, GCC only shows the message on the second line instead of the "#pragma message" part. The note itself displays the same for both though.
```[tav@myriad test]$ g++ -o Main Main.cpp Test.cpp 
Main.cpp:4:17: note: #pragma message: warning: hi
 #pragma message "warning: hi"
                 ^~~~~~~~~~~~~
[tav@myriad test]$ g++ -o Main Main.cpp Test.cpp 
Main.cpp:5:3: note: #pragma message: warning: hi
   "warning: hi"
   ^~~~~~~~~~~~~
```

Reply:
GCC doesn't even hit the pragma message (it uses the #warning directive instead), so I'm actually more concerned with what MSVC does with it.

Reply:
I put the directives on the same line and put "// clang-format off" and "// clang-format on" around them.